frontend/simple/js/invites.js

```diff
@@ -0,0 +1,31 @@
+import * as Events from '../../../shared/events'
+import backend from './backend'
+
+export const createInviteMail = async function (mailbox, groupName, groupId) {
+  const latestMailbox = await backend.latestHash(mailbox)
+  const sentDate = new Date().toString()
+  // create the invite record to the users' mailbox contract
+  return new Events.HashableMailboxPostMessage(
+    {
+      from: groupName,
+      headers: [groupId],
+      messageType: Events.HashableMailboxPostMessage.TypeInvite,
+      sentDate
+    },
+    latestMailbox
+  )
+}
+
+export const createInviteToGroup = async function (inviteHash, memberName, groupId) {
+  const latest = await backend.latestHash(groupId)
+  const sentDate = new Date().toString()
+  // create record of the invitation in the group's contract
+  return new Events.HashableGroupRecordInvitation(
+    {
+      username: memberName,
+      inviteHash: inviteHash,
+      sentDate
+    },
+    latest
+  )
+}
```

Comment:
FYI, these kinds of functions are a good candidate for an SBP namespace (along with a bunch of other similar functions). Can you put a TODO comment at the top to convert to SBP?

Reply:
sure! I'll look into this, seems like a good candidate to try my hand with SBP :)